src/js/Background/background.ts

```diff
@@ -52,44 +51,45 @@ browser.runtime.onMessage.addListener((
         return;
     }
 
-    (async function(): Promise<void> {
-        try {
-            await Promise.all([IndexedDB.init(), SettingsStore.init()]);
-
-            let response: any = undefined;
-            let handlers: MessageHandlerInterface[] = [
-                new AugmentedSteamApi(),
-                new SteamCommunityApi(),
-                new InventoryApi(),
-                new ITADApi(),
-                new SteamStoreApi(),
-                new UserNotesApi(),
-                new CacheApi()
-            ];
-
-            for (let handler of handlers) {
-                response = await handler.handle(message);
-                if (response !== Unrecognized) {
-                    break;
+    return new Promise((resolve, reject) => {
```

Comment:
Isn't it enough just returning that async function call? That's already a promise, unless Chrome handles things differently, that's just wrapping a promise in promise.